modules/opensearch/opensearch.client.inc

```diff
@@ -235,6 +235,10 @@ function opensearch_do_search($query, $page = 1, $results_per_page = 10, $option
     }
     $request->userDefinedBoost = $uboosts;
   }
+  
+  if (isset($options['fullTextQuery'])) {
+    $request->fullTextQuery = $options['fullTextQuery'];
```

Comment:
Maybe a short comment explaining why this is needed?

Reply:
@Arni How did you end up resolving this? Looking at the code, I can't see it handled anywhere.

Reply:
We use the search query provided with search object instead. The changes made by CQLdoctor are easy to remove for simple queries and the module isn't designed to handle more complex queries.